hyp3_rtc_gamma/__main__.py

```diff
@@ -346,20 +352,18 @@ def process_rtc_gamma(cfg, n):
         else:
             raise Exception('Unrecognized: ' + in_granule)
 
-        product = find_product(cfg['workdir'])
-        if not os.path.isdir(product):
-            log.info('PRODUCT directory not found: ' + product)
+        product_dir = find_product_dir(cfg['workdir'])
+        if not os.path.isdir(product_dir):
+            log.info('PRODUCT directory not found: ' + product_dir)
             log.error('Processing failed')
             raise Exception("Processing failed: PRODUCT directory not found")
         else:
             if extra_arg_is(cfg, "include_dem", "no"):
-                find_and_remove(product, '_dem.tif')
+                find_and_remove(product_dir, '_dem.tif')
             if extra_arg_is(cfg, "include_inc", "no"):
-                find_and_remove(product, '_inc_map.tif')
+                find_and_remove(product_dir, '_inc_map.tif')
 
-            out_name = build_output_name(g, cfg['workdir'], opts_str + cfg['suffix'])
-            if "STD_L0_F" in out_name:
-                out_name = out_name.replace("_L0", "")
+            out_name = find_product_name(product_dir)
```

Comment:
need to address non-sentinel products, I think the other use case is PALSAR via rtc_legacy.py

Reply:
I think other products will need to be addressed separately; there is a bit more work to do from what I can tell to get them supported in the containers (missing dependencies). Maybe just make an issue for this

Reply:
Yeah, should just get confirmation from Kirk that things are already broken in production, in which case this change won't make things any worse.